src/commands/sarif/renderer.ts

```diff
@@ -45,6 +47,7 @@ export const renderSuccessfulCommand = (
 ) => {
   let fullStr = ''
   fullStr += chalk.green(`${ICONS.SUCCESS} Uploaded ${fileCount} files in ${duration} seconds.\n`)
+  fullStr += chalk.green(`${ICONS.INFO}  Results available on ${getBaseUrl()}ci/code-analysis\n`)
```

Comment:
Could we make the link more specific with some query params? For example, since we give `--service` to the command, the link could at least filter on the service.

Example for JUnit uploads: https://github.com/DataDog/datadog-ci/blob/4e722da522276da8bf1089087d984dee8b74b837/src/commands/junit/renderer.ts#L46-L69

Reply:
I can and thought about it. The reason I did not is because the page is still significantly changing and I wanted to prevent linking to a page that has URL params that may change quickly in the near future. But if you think this is necessary, I can change this.

Reply:
Looked again at this and this is definitely something I do not want to do.

1. First, the results page for static analysis/sca are changing a lot. We risk to expose an outdated URL in the future.
2. The URL parameters use the repository id and calculating the repository ID from the CLI will lead to 404. We already have multiple issues relatd to repository ID and I really do not want to add more.

If we want to handle this correctly, we should let the frontend/backend accepts a repository URL and have the backend transforms to a consistent repository id but that is def outside the scope of this PR.

Reply:
Ok 👌